src/Microsoft.DocAsCode.Build.Engine/CompilePhaseHandlerWithIncremental.cs

```diff
@@ -88,6 +89,7 @@ private void PostHandle(List<HostService> hostServices)
                 h.SaveIntermediateModel(IncrementalContext);
             }
             IncrementalContext.UpdateBuildVersionInfoPerDependencyGraph();
+            SaveContextInfo(hostServices);
```

Comment:
just find that some article might build incrementally in `Compile` phase while non-incrementally in `Link` phase, this way we have to have phase data